packages/react-cli/src/commands/fetch-configs.ts

```diff
@@ -0,0 +1,29 @@
+import { Command } from '@oclif/command';
+import { ensureDependencies } from '../tasks/ensure-dependiencies';
+import Listr from 'listr';
+import { downloadTSConfigFiles } from '../tasks/download-ts-config';
+import { exec } from '../utils/shell';
+
+export class FetchConfigsCommand extends Command {
+  static description = 'Fetches config files for linting and typescript projects';
+
+  static examples = ['$ react fetch-configs'];
+
+  async run() {
+    await ensureDependencies();
+
+    let path = './';
```

Comment:
what is this path relative to?

Reply:
current working directory. could probably rename the variable to that